services/checks/checks.go

```diff
@@ -390,8 +393,6 @@ func (s *Service) run(ctx context.Context, intervalGroup check.Interval, checkNa
 		return errors.WithStack(err)
 	}
 
-	s.CollectChecks(ctx)
```

Comment:
What was the reason to move it out from here?

Reply:
`runChecksLoop()` is called for each interval (standard, rare, frequent) so we are collecting all the checks three times.

Moving it out means that we only download checks once per execution. Also, now  `run` only cares about running the checks, not downloading them

Reply:
@idoqo But nothing changed. You still download checks on each group tick.

Reply:
it was reverted in this PR https://github.com/percona/pmm-managed/pull/1082/commits/b56ea0587d005eba95b17f0efde702602a35c6cd

Reply:
Yeah yeah - forgot to update this comment but I ended up reverting it (sort of).

I realized it makes sense to re-download the checks on each interval because `Run()` only gets called when managed starts up, so if for instance, a user moves from paid to registered (or vice-versa), the checks we downloaded on startup remains in the cache.

So yeah, the only thing different now is moving `CollectChecks()` out of the run method.